contracts/interfaces/IProxyCreationCallback.sol

```diff
@@ -0,0 +1,12 @@
+
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.0;
+
+interface IProxyCreationCallback {
+    function proxyCreated(
+        address proxy,
+        address _singleton,
+        bytes calldata initializer,
+        uint256 saltNonce
+    ) external;
+}
```

Comment:
I believe this interface exists in the @gnosis.pm/safe-contracts npm package that we are importing, so probably no need to create it manually